megamek/src/megamek/common/IBomber.java

```diff
@@ -52,6 +52,11 @@ default int getMaxBombSize() {
      * Sets the count of each bomb to zero
      */
     void clearBombChoices();
+
+    /**
+     * @return The load for bombs currently on unit
+     */
+    int getBombLoad(int t);
```

Comment:
I assume the name comes from the "// get bomb load" comments but since this method has found its way to the IBomber interface I'd prefer a name that actually says what it does, like reduceMPForBombLoad or something. Also the comment here might preferably say what the parameter is and what the method is supposed to return.